docs/book/v3/migration/v2-to-v3.md

```diff
@@ -131,6 +131,21 @@ Additionally, `$options['pattern']` _must_ be provided at construction time or a
 
 Exceptions for invalid or empty patterns are now thrown during construct rather than when the filter is invoked.
 
+#### `SeparatorToDash`
+
+The constructor now only accepts an associative array of [documented options](../word.md#separatorToDash).
```

Comment:
There aren't really any changes here because there is no constructor, or options. Except the class hierarchy changes ofc

Reply:
Thanks. Just to double check before I change this
I think this one does have changes here?
https://github.com/laminas/laminas-filter/pull/193/files#diff-c75996343cb3fe18c4af5cd57d67b8e5f7991b79e866333210808d8dbfab3709R21-R24

Reply:
My mistake! You're absolutely right :)